staging/traefik-forward-auth/templates/clusterrole.yaml

```diff
@@ -35,3 +35,12 @@ rules:
 - apiGroups: [""]
   resources: ["services"]
   verbs: ["get", "list", "watch"]
```

Comment:
you didn't touch this by why is this listed differently than the others in this yaml?

Reply:
@jr0d could you bring the above to the same style as what you added in? please and thanks!

Reply:
👍